README.md

```diff
@@ -24,3 +24,11 @@ Atril Requirements to Run Tests
 ==================================================
 * [python-dogtail](https://gitlab.com/dogtail/dogtail)
 * [python-pyatspi2](https://download.gnome.org/sources/pyatspi/)
+* git installed on your system
+
+Atril Compilation
+==================================================
+* git submodule init
+* git submodule update
```

Comment:
Can we use `git submodule update --remote --recursive`, please?

Reply:
Same for other PRs where you want to add it :)

Reply:
Ok, no problem.

Reply:
Done.